planner/core/common_plans.go

```diff
@@ -664,3 +665,33 @@ func (e *Explain) prepareTaskDot(p PhysicalPlan, taskTp string, buffer *bytes.Bu
 		buffer.WriteString(pipelines[i])
 	}
 }
+
+func sizeOfPlan(p Plan) (sz int) {
```

Comment:
it seems not enough room because of we mix `noChunksUsed` into `computeHashImpl`?

Reply:
I agree that the initial capacity of the map may not be enough. Can we know the exact number in advance when caching the plan?